.eslintrc.js

```diff
@@ -4,9 +4,7 @@ module.exports = {
     jest: true
   },
   rules: {
-    'no-proto': 0,
-    strict: 0,
-    'max-len': 0,
+    'no-proto': 0
```

Comment:
why did you remove this rule?

Reply:
I don't know, because I didn't even open this file. I opened and changed only src/splitInteger.test.js